server/config.ts

```diff
@@ -96,6 +96,6 @@ export default {
   domain: get('INGRESS_URL', 'http://localhost:3000', requiredInProduction),
   dpsHomeUrl: get('DPS_URL', 'http://localhost:3000/', requiredInProduction),
   featureToggles: {
-    editGoalsEnabled: Boolean(get('EDIT_GOALS_ENABLED', false)),
+    // someToggleEnabled: Boolean(get('SOME_TOGGLE_ENABLED', false)),
```

Comment:
I left this as an example for future use, rather than an empty `featureToggles` object which might not have been so obvious how to use in the future.

Reply:
Nice idea 👍